docs/osctl/osctl_cluster.md

```diff
@@ -20,7 +20,7 @@ A collection of commands for managing local docker-based clusters
       --context string       Context to be used in command
   -e, --endpoints strings    override default endpoints in Talos configuration
   -n, --nodes strings        target the specified nodes
-      --talosconfig string   The path to the Talos configuration file (default "/root/.talos/config")
+      --talosconfig string   The path to the Talos configuration file (default "/home/user/.talos/config")
```

Comment:
these are defaults with `make docs` now, hopefully we won't have any more changes there